.github/workflows/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# Default fallback
+*    @abcxyz/breakglass
+
+# products owns all files
+*    @abcxyz/infrastructure-team
```

Comment:
Add empty line at the end

Reply:
Throughout

Reply:
Oh, this is also the wrong location. I've already added CODEOWNERS earlier.

Reply:
I follow the format from jvs repo actually, I guess it probably is in the wrong place. I am gonna leave it out in this PR. 

https://github.com/abcxyz/jvs/blob/c74e145411363aed1d245d5530ed92f246df1abe/.github/CODEOWNERS,